src/rt/test/systematic.h

```diff
@@ -303,8 +313,8 @@ namespace Systematic
 #endif
 
   private:
-    std::ostream& o;
-    bool first = true;
+    std::ostream* o;
+    bool first ;
```

Comment:
Is there a reason not to default these members to `nullptr` and `true`?

Reply:
It lead to a constructor and codegen.